### DISCUSSION TITLE
Phising website

### DISCUSSION THREAD

**Discussion by denisanjayaaa** [vercel/community#3485] [Help] [answered]
### Summary

https://iddanakagtt1.vercel.app/

### Example

https://iddanakagtt1.vercel.app/

### Steps to Reproduce

https://iddanakagtt1.vercel.app/

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this @denisanjayaaa. I let the team know about this post, but please also [submit a case](https://vercel.com/help#issues) under the `Abuse` problem area. That will help our team keep track of it while we investigate. If you are not able to use the form, you can email [abuse@vercel.com](mailto:abuse@vercel.com).

Please provide as much relevant information as possible to help with investigation. 🙏

**Reply by wahluf** [reply]
hey, how mechanism if we want to send phishing project?.. 2 week ago i email to abuse@vercel.com and still no action from your team. please, help to quick response.

**Reply by wahluf** [reply]
@amyegan

**Reply by denisanjayaaa** [reply]
Hello, would you like me to provide some tips and steps for it? I'd be happy to assist you. Contact me @denisanjayaaa

**Reply by wahluf** [reply]
hey @denisanjayaaa, thanks for your inisiate. i try to contact you from telegram but your change username. can you give me ur latest username?

**Reply by denisanjayaaa** [reply]
hello @wahluf , you can contact me at https://t.me/itsmedexproject
